src/components/debug/DebugCreateStrategy.tsx

```diff
@@ -25,9 +26,14 @@ const TOKENS = FAUCET_TOKENS.map((tkn) => ({
   symbol: tkn.symbol,
 }));
 
-TOKENS.push({ address: config.tokens.ETH, decimals: 18, symbol: 'ETH' });
+const gasToken = config.network.gasToken;
+TOKENS.push({
+  address: gasToken.address,
+  decimals: gasToken.decimals,
+  symbol: gasToken.symbol,
+});
 
-const spender = config.carbon.carbonController;
+const spender = networkConfig.carbon.carbonController;
```

Comment:
I dont think we need both configs, dont we have the carbon controller address in the config as well?

Reply:
Agreed - and to add to it, networkConfig also gets the `carbonControllerAddress` and the `voucherContractAddress` from localStorage, but these values are never set, which means we might as well remove it and use the one from config only. Replacing everywhere networkConfig is used.